gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/traverser/O_Traverser.java

```diff
@@ -23,6 +23,7 @@
 import org.apache.tinkerpop.gremlin.process.traversal.traverser.util.AbstractTraverser;
```

Comment:
There is no `hashCode()` method for this class either like `B_O_...`. Again, its just `return t.hashCode()`